snap/quota/quota.go

```diff
@@ -153,6 +221,253 @@ func (grp *Group) SliceFileName() string {
 	return buf.String()
 }
 
+// groupQuotaStats contains information about current quotas of a group
+// and is used by getGroupQuotaStats to contain this information
+type groupQuotaStats struct {
+	memoryLimit quantity.Size
+	memoryUsed  quantity.Size
```

Comment:
is Used here more like already reserved/allocated to groups?

Reply:
I'll change to 'reserved', makes more sense